src/client.jsx

```diff
@@ -34,12 +37,42 @@ if (
 
 const client = createApolloClient(i18nInstance.language);
 
+const LanguageWrapper = () => {
+  const { i18n } = useTranslation();
+  const history = useHistory();
+  const client = useApolloClient();
+  const [lang, setLang] = useState(basename);
+  const firstRender = useRef(true);
+  initializeI18n(i18n, client, history);
+
+  useEffect(() => {
+    if (firstRender.current) {
+      firstRender.current = false;
+      return;
+    }
+    const supportedLanguages = i18n.options.supportedLngs;
+    const regex = new RegExp(supportedLanguages.map(l => `/${l}/`).join('|'));
+    const paths = window.location.pathname.replace(regex, '').split('/');
+    const { search } = window.location;
+    const p = paths.slice().join('/');
+    const test = p.startsWith('/') ? p : `/${p}`;
+    history.replace(`/${i18n.language}${test}${search}`);
+    setLang(i18n.language); // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [i18n.language]);
+
+  return (
+    <BrowserRouter basename={lang} key={lang}>
+      <App />
+    </BrowserRouter>
```

Comment:
Her får du en BrowserRouter inni en BrowserRouter, men det gjør kanskje ingenting?

Reply:
Jeg tror ikke det er anbefalt, men det er støtte for det i React Router V5. I V6 vil dette ikke fungere, men da kommer det relativ pathing som forhåpentligvis vil la oss gjøre ting på en mer "riktig" måte.